packages/frontend/src/components/common/Filters/FilterInputs/DateFilterInputs.tsx

```diff
@@ -201,7 +208,7 @@ const DateFilterInputs = <T extends ConditionalRule = DateFilterRule>(
                                         : [
                                               formatTimestamp(
                                                   value,
-                                                  TimeFrames.MILLISECOND,
+                                                  TimeFrames.SECOND,
```

Comment:
Fixes this issue https://lightdash.slack.com/archives/C02GQKJK84Q/p1706005922819219

Reply:
This is the full stacktrace. 

It seems DateTimePicker doesn't like the `onChange`d milisecond SSS format , so it cascades down to HiddenDatesInput and breaks. It also feels this is not handled properly by the mantine component. 

![image](https://github.com/lightdash/lightdash/assets/1983672/51d24e66-00ef-43c2-992d-2e41be172a10)

Reply:
too bad. I see.. thanks!